README.md

```diff
@@ -1,47 +1,51 @@
+## Table of contents
 
-* :satellite: &nbsp; [Updating the Documentation](https://github.com/LivePersonInc/developers-community#updating-the-documentation)
-* :ghost: &nbsp; [Hiding Files](https://github.com/LivePersonInc/developers-community#hiding-files)
-* :hammer: &nbsp; [Building the Site Locally](https://github.com/LivePersonInc/developers-community#building-the-site-locally)
-* :clipboard: &nbsp; [Template](https://github.com/LivePersonInc/developers-community#template)
-* :scroll: &nbsp; [Licensing](https://github.com/LivePersonInc/developers-community#licensing)
+1. 📡 [Updating the documentation](https://github.com/LivePersonInc/developers-community#updating-the-documentation)
+2. 📝 [Notes on content and code](https://github.com/LivePersonInc/developers-community#notes-on-content-and-code)
+3. 👻 [Hiding files](https://github.com/LivePersonInc/developers-community#hiding-files)
+4. 🔨 [Building the site locally](https://github.com/LivePersonInc/developers-community#building-the-site-locally)
+5. 📋 [Template](https://github.com/LivePersonInc/developers-community#template)
+6. 📜 [Licensing](https://github.com/LivePersonInc/developers-community#licensing)
 
-### Updating the Documentation
+## Updating the documentation
 
-All pages on the site correspond to a Markdown file (.md) which can be found inside `pages/Documents`. 
-To update a file, please branch off of the `master` branch, edit the file in question and create a Pull Request **back to the master branch**.
+All pages on the site correspond to a Markdown file (.md) which can be found inside `pages/Documents`. To update a file, please branch off of the `master` branch, edit the file in question, and create a pull request back to the master branch.
 
-#### Committing Changes to the Developers Site
+### Committing changes to the site
 
-Before making any commits, please make sure to read the Updating/Creating Headers section. There is now a Git precommit hook that makes sure you follow the rules on Markdown file creation. This hook will run on every commit and deny commits if they fail the test. The errors will be outputed to `./_scripts/docOutputError.log`. If you are adding new content, please make sure you are updating the content in documentsupdated.yaml file. Our tests will use the documents updated yaml file as the source of truth. So make sure your header naming structure matches the documentsupdated.yaml.
+Before making any commits, please make sure to read the _Updating and creating headers_ section. There is now a Git precommit hook that ensures you follow the rules on Markdown file creation. This hook will run on every commit and deny commits if they fail the test. The errors will be outputted to `./_scripts/docOutputError.log`. If you are adding new content, please make sure you are updating the content in the documentsupdated.yaml file. Our tests will use that YAML file as the source of truth, so make sure your header naming structure matches the documentsupdated.yaml.
```

Comment:
(@lkisoslive, all changed except for the last comment—it seems there is some context in this sentence, but please clarify and we can update further.)